addons/web/static/src/views/fields/properties/property_definition_selection.js

```diff
@@ -198,6 +229,36 @@ export class PropertyDefinitionSelection extends Component {
         options.splice(optionIndex, 1);
         this.props.onOptionsChange(options);
     }
+
+    /**
+     * Move an option after an other one.
+     *
+     * @param {string} from, the option to move
+     * @param {string} to, the target option
+     *      (null if we move the option at the first index)
+     */
+    onOptionMoveTo(from, to) {
+        const options = this.optionsVisible;
+        const fromIndex = options.findIndex((option) => option[0] == from);
+        let toIndex = options.findIndex((option) => option[0] == to);
```

Comment:
Would add a small comment mentioning that if to is null, we get -1, which is intended.

(Had to read the doc of findIndex to understand ^^)

Reply:
indeed, not very obvious :)